exonum-java-binding/cryptocurrency-demo/src/test/java/com/exonum/binding/cryptocurrency/transactions/CreateWalletTxTest.java

```diff
@@ -67,51 +74,47 @@ void constructorRejectsNegativeBalance() {
 
   @Test
   @RequiresNativeLibrary
```

Comment:
I'm not sure that this annotation is still needed to be applied with tests which use `TestKit`. Of course, it adds a description of the test. From another hand, `TestKit` doc says that it initializes Exonum blockchain instance (which requires the native library to be loaded). So I propose to apply `@RequiresNativeLibrary` at `TestKitExtension` and remove the annotation in tests. 
@dmitry-timofeev @MakarovS  WDYT?

Reply:
`@RequiresNativeLibrary` is our internal annotation for tagging (and, possibly, subsequent filtering — we used to use it to skip such tests on Linux). It has no other effects. 

I don't think the testkit extension (which is suppossed to be used by other) shall bring our tags for client tests (= auto-tag them).